@navikt/core/react/src/chat/Chat.tsx

```diff
@@ -90,24 +102,26 @@ export const Chat = forwardRef<HTMLDivElement, ChatProps>(
         className={cl(
           "navds-chat",
           className,
-          `navds-chat--${position} navds-chat--top-text-${
-            toptextPosition ?? position
-          }`
+          `navds-chat--${position}`,
+          `navds-chat--top-text-${toptextPosition ?? position}`,
+          `navds-chat--size-${size}`
```

Comment:
I andre komponenter har vi bare satt size-prop direkte som modifier, f.eks i button
```ts
`navds-button--${variant}`,
`navds-button--${size}`,
```

Reply:
Dette henger nok igjen fra et tidligere utkast der default size var "default" i stedet for "medium", slik at "navds-chat--default" ville blitt veldig generisk. Kan fjerne "size"-prefiksen.